### PR TITLE
V1

### PR DESCRIPTION
**Pull request by Oba-One**

### CODE REVIEW

File: packages/contracts/src/components/PhaseComponent.sol [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 
 import "std-contracts/components/Uint256Component.sol";
-import { PhaseEnum } from "libraries/MSTypes.sol";
+import {PhaseEnum} from "libraries/MSTypes.sol";
 
 uint256 constant ID = uint256(keccak256("component.Position"));
 
-contract PositionComponent is Uint256Component {
-  constructor(address world) Uint256Component(world, ID) {}
+contract PhaseComponent is Uint256Component {
```
Comment (junaama) [inline packages/contracts/src/components/PhaseComponent.sol:9]: why is phase replacing position?

File: packages/contracts/src/components/ZoneComponent.sol [hunk 2]
```diff
@@ -4,9 +4,9 @@ pragma solidity >=0.8.0;
 import "std-contracts/components/Uint256Component.sol";
 import { ZoneEnum } from "libraries/MSTypes.sol";
 
-uint256 constant ID = uint256(keccak256("component.Position"));
+uint256 constant ID = uint256(keccak256("component.Zone"));
 
-contract PositionComponent is Uint256Component {
+contract ZoneComponent is Uint256Component {
```
Comment (junaama) [inline packages/contracts/src/components/ZoneComponent.sol:9]: why is zone replacing position?

File: packages/contracts/src/libraries/LibCardMove.sol [hunk 3]
```diff
@@ -16,6 +16,4 @@ import { Uint32Component } from "std-contracts/components/Uint32Component.sol";
  * @param
  */
 
-library LibCardMove {
-
-}
+library LibCardMove {}
```
Comment (junaama) [inline packages/contracts/src/libraries/LibCardMove.sol:19]: unimplemented

**Review by junaama — COMMENTED** [review]
most of the logic still isn't there @mrhouzlane also we need more tests to ensure data is being updated properly

File: packages/contracts/src/components/ZoneComponent.sol (hunk 2 above)
Comment (Oba-One) [reply packages/contracts/src/components/ZoneComponent.sol:9]: zone is not replacing position I copied the position component but forgot to change the name to zone. Zone refers to the 4 spots on a location and is tied to the character.

File: packages/contracts/src/components/PhaseComponent.sol (hunk 1 above)
Comment (Oba-One) [reply packages/contracts/src/components/PhaseComponent.sol:9]: typo

**Pull request merged**